### PR TITLE
Use 1.26.0 instead of beta for cluster testing

### PR DESCRIPTION
**Pull request by jacob-delgado**

### CODE REVIEW

File: prow/config/jobs/istio.yaml [hunk 1]
```diff
@@ -346,7 +346,7 @@ jobs:
       - entrypoint
       - prow/integ-suite-kind.sh
       - --node-image
-      - gcr.io/istio-testing/kind-node:v1.26.0-beta.0
+      - gcr.io/istio-testing/kind-node:v1.26.0
       - test.integration.kube
     requirements: [kind]
     modifiers: [hidden] # Hidden until job is stable
```
Comment (jacob-delgado) [inline prow/config/jobs/istio.yaml:352]: not sure if we should unhide this at this point in time

**Comment by jacob-delgado** [comment]
cc @howardjohn need a new image pushed to gcr from https://hub.docker.com/layers/kindest/node/v1.26.0/images/sha256-7998effe843cbcb88bc6876a142437e7bccf6d77c5a928dd2325f2ff6fee6f60?context=explore

**Review by howardjohn — COMMENTED** [review]
ill build this image

**Review by howardjohn — APPROVED** [review]
build should be done in ~10min then we can unhold

**Comment by ericvn** [comment]
@howardjohn I don't see the image at https://console.cloud.google.com/gcr/images/istio-testing/GLOBAL/kind-node

**Comment by howardjohn** [comment]
sorry image failed, building again

**Comment by howardjohn** [comment]
should be good now

**Pull request merged**